src/app/pages/activities-nearby/activities-nearby.component.scss

```diff
@@ -1,3 +1,15 @@
+@import url('https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap');
```

Comment:
import fonts isnt necesary here, because its loaded as a link in the index.html from google fonts